### DISCUSSION TITLE
How to Make a Note Block (Callout / Admonition) Appear After an `.incremental` Block on RevealJS

### DISCUSSION THREAD

**Discussion by RoyiAvital** [quarto-dev/quarto-web#281] [Q&A]
Assume we have a slide defined as:

```
## The Slide Title

::: {.incremental}
 - Bullet A
 - Bullet B
 - Bullet C
::: 

:::{.callout-note}
My very not important note...
:::
```

I want the not to appear on click after `Bullet C`.
How can I do this?

**Comment by andrewpbray** [comment]
The easiest way is to use `. . .` between the list and the callout note.

```
## The Slide Title

::: {.incremental}
 - Bullet A
 - Bullet B
 - Bullet C
::: 

:::{.callout-note}
My very not important note...
:::
```

It's mentioned briefly here: https://quarto.org/docs/presentations/revealjs/#incremental-lists

**Reply by RoyiAvital** [reply]
OK, Is there a more complex / general way?

Thanks for that, I missed that for some reason.

**Comment by andrewpbray** [comment]
The alternative approach that I know of is to explicitly put it in a fragment block:
```
## The Slide Title

::: {.incremental}
 - Bullet A
 - Bullet B
 - Bullet C
::: 

. . .

:::{.fragment}
:::{.callout-note}
My very not important note...
:::
:::
```

If you look at the HTML that results from the two methods, you'll find they're the same:

<img width="925" alt="Screen Shot 2022-07-27 at 8 50 27 AM" src="https://user-images.githubusercontent.com/1627362/181313099-fcc82428-c0f5-4013-81a0-264c47f82cba.png">

You also learn here that `{.incremental}` also relied on the `fragment` class.

**Reply by RoyiAvital** [reply]
But you used the `. . .` in this as well.
I was asking because just adding `:::{.fragment}` did not work for me.

**Reply by andrewpbray** [reply]
Whoops, that was a typo. I removed it.

Give it a try.
